test/format/format_config.c

```diff
@@ -1623,6 +1627,40 @@ config_tiered_storage(void)
         config_single(NULL, "tiered_storage.flush_frequency=0", true);
 }
 
+/*
+ * config_disagg_storage --
+ *     Disaggregated storage configuration.
+ */
+static void
+config_disagg_storage(void)
+{
+    const char *page_log;
+
+    page_log = GVS(DISAGG_PAGE_LOG);
+
+    /*
+     * FIXME-WT-9934 If we ever allow tiered storage to be run only locally but with switching
```

Comment:
Nitpick, tiered storage comments/fixmes lifted verbatim in this section.

Reply:
fixed